src/Model/DeviceItem.ts

```diff
@@ -4,6 +4,7 @@
 import { Command, QuickPickItem, TreeItem } from "vscode";
 
 export class DeviceItem extends TreeItem implements QuickPickItem {
+    public readonly label: string;

```

Comment:
defined not used?

Reply:
DeviceItem needs to implement label property in QuickPickItem